com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -523,8 +623,58 @@ public void Execute(ScriptableRenderContext context, ref RenderingData rendering
                     // is because this need to be called for each eye in multi pass VR.
                     // The side effect is that this will override some shader properties we already setup and we will have to
                     // reset them.
-                    context.SetupCameraProperties(camera);
-                    SetCameraMatrices(cmd, ref cameraData, true);
+                    if (cameraData.renderType == CameraRenderType.Base)
+                    {
+                        context.SetupCameraProperties(camera);
+                        SetCameraMatrices(cmd, ref cameraData, true);
+                    }
+                    else
+                    {
+                        // TODO: Remove after the testing
+                        context.SetupCameraProperties(camera);
+                        SetCameraMatrices(cmd, ref cameraData, true);
+
+                        // TODO: Remove after the testing
+                        TestCameraProperties.SampleCameraPropertiesOld(cmd);
+
+                        // Set new properties
+                        SetPerCameraShaderVariables(cmd, ref cameraData); // TODO: Remove after the testing
+                        SetCameraMatrices(cmd, ref cameraData, true);
+                        SetPerCameraClippingPlaneProperties(cmd, in cameraData); // TODO: Move this is to SetCameraMatrices, once we get rid of context.SetupCameraProperties
+                        SetPerCameraBillboardProperties(cmd, ref cameraData);
+
+                        // TODO: Find out if this is needed
+                        /*#if GFX_SUPPORTS_SINGLE_PASS_STEREO
```

Comment:
Need someone from XR team to cofirm is this needed?

Reply:
I don't think they are needed anymore.